src/internal/Modal.spec.js

```diff
@@ -51,6 +51,99 @@ describe('<Modal />', () => {
       assert.strictEqual(modal.is('div'), true, 'should be a div');
       assert.strictEqual(modal.hasClass(classes.modal), true, 'should have the modal class');
     });
+
+    describe('handlers', () => {
+      let instance;
+
+      before(() => {
+        instance = wrapper.instance();
+      });
+
+      describe('focus', () => {
+        before(() => {
+          instance.modal = spy();
+          instance.modal.lastChild = spy();
+          instance.modal.lastChild.setAttribute = spy();
+          instance.modal.lastChild.focus = spy();
+        });
+
+        describe('modalContent has tabIndex attribute', () => {
+          let consoleError;
+          let consoleErrorSpy;
+
+          before(() => {
+            instance.modal.lastChild.hasAttribute = stub().returns(true);
+            consoleError = console.error; // eslint-disable-line no-console
+            consoleErrorSpy = spy();
+            console.error = consoleErrorSpy; // eslint-disable-line no-console
```

Comment:
I have to say this isn't a clear case of monkey patching as its done for one call of one function. I agree finding a systemic way of testing a call to warning should be implemented, though that would seem to be out of scope for this project.